imagetagger/imagetagger/images/models.py

```diff
@@ -24,6 +29,17 @@ def path(self):
     def relative_path(self):
         return os.path.join(self.image_set.path, self.filename)
 
+    def delete(self, using=None, keep_parents=False):
```

Comment:
If you do not care about the arguments, it might be better to use (self, *args, **kwargs) as signature for better forward-compatibility.